### PR TITLE
LPD-20257 | Master

### PR DESCRIPTION
**Pull request by interaminense**

### CODE REVIEW

**Comment by interaminense** [comment]
This PR depends on this one to be merged: https://github.com/liferay-frontend/liferay-portal/pull/4061

File: modules/apps/analytics/analytics-web/src/main/resources/META-INF/resources/dynamic_include/top_head.jsp [hunk 1]
```diff
@@ -21,63 +21,102 @@
...
-		runMiddlewares();
-
-		Analytics.send('pageViewed', 'Page');
-
-		<c:if test="<%= GetterUtil.getBoolean(PropsUtil.get(PropsKeys.JAVASCRIPT_SINGLE_PAGE_APPLICATION_ENABLED)) %>">
-			Liferay.on('endNavigate', (event) => {
-				Analytics.dispose();
-
-				var groupId = themeDisplay.getScopeGroupIdOrLiveGroupId();
+	function <portlet:namespace />checkLiferayDXPConsent() {
```
Comment (marcellustavares) [inline modules/apps/analytics/analytics-web/src/main/resources/META-INF/resources/dynamic_include/top_head.jsp:24]: can we rename to `checkPerformanceCookieConsent` ?

File: modules/apps/analytics/analytics-web/src/main/resources/META-INF/resources/dynamic_include/top_head.jsp [hunk 2]
```diff
@@ -21,63 +21,102 @@
...
+			Liferay.Util.Cookie.TYPES.PERFORMANCE
+		);
+
+		return (
+			!analyticsCookiesConsentMode ||
+			(analyticsCookiesConsentMode && performanceCookieEnabled === 'true')
+		);
+	}
+
+	function <portlet:namespace />validateConsentCookie() {
```
Comment (marcellustavares) [inline modules/apps/analytics/analytics-web/src/main/resources/META-INF/resources/dynamic_include/top_head.jsp:35]: can we rename to `isPerfomanceCookieConsented`?

**Review by marcellustavares — CHANGES_REQUESTED** [review]
@interaminense

**Comment by marcellustavares** [comment]
@interaminense, I've made some minor function name request changes. Let me know if they make sense. Thanks

**Comment by marcosapmf** [comment]
resent to https://github.com/liferay-ac/liferay-portal/pull/1275

**Pull request closed without merging**